images/ansible/roles/kubernetes/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+kubernetes_version: ""
+kubernetes_cni_version: ""
```

Comment:
Does this behave correctly with the defaults here? It appears this may fail, since we are passing the value into the kube_platform_version filter.